daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -1534,33 +1538,61 @@ private[lf] final class Compiler(
     ref -> SDefinition(withLabel(ref, unsafeCompile(method.value)))
   }
 
-  private[this] def compileCreate(
+  @nowarn("msg=parameter value tokenPos in method compileCreateBody is never used")
```

Comment:
probably a silly question but why do we pass it in if we don’t need it?

Reply:
I guess so that `topLevelFunction2` works without having to wrap it? I think I’d take the lambda wrapper that drops the argument but happy to leave it up to you.

Reply:
That and tradition mostly. There's a `compileFetchBody` that is very similar. But I'll try to remove the tokenPos entirely.

Reply:
if `compileFetchBody` is similar then it’s perfectly fine to keep it like this in this PR.